apps/andi/lib/andi_web/input_schemas/metadata_form_schema.ex

```diff
@@ -20,6 +20,7 @@ defmodule AndiWeb.InputSchemas.MetadataFormSchema do
     field(:dataName, :string)
     field(:dataTitle, :string)
     field(:datasetId, :string)
+    field(:userId, :string)
```

Comment:
I would rename this to `user_id` to stay consistent with the dataset changeset association